app/views/content_items/consultation.html.erb

```diff
@@ -0,0 +1,232 @@
+    </h2>
+  </div>
+<% elsif @content_item.pending_final_outcome? %>
+  <div class="consultation-notice">
+    <h2>
+      We are analysing your feedback
+    </h2>
+    <p>Visit this page again soon to download the outcome to this public&nbsp;feedback.</p>
+  </div>
+<% elsif @content_item.final_outcome? %>
+  <div class="consultation-notice">
+    <h2>
+      This consultation has concluded
+    </h2>
+  </div>
+
+  <% if @content_item.final_outcome_documents? %>
+    <div class="grid-row sidebar-with-body">
+      <div class="column-third">
+        <h1 class="section-title" id="final-outcome-documents-title">
+          Download the full outcome
+        </h1>
+      </div>
+      <div class="column-two-thirds" aria-labelledby="final-outcome-documents-title">
+        <%= render 'govuk_component/govspeak',
+            content: @content_item.final_outcome_documents,
+            direction: page_text_direction %>
+      </div>
+    </div>
+  <% end %>
+
+  <div class="grid-row sidebar-with-body">
+    <div class="column-third">
+      <h1 class="section-title" id="final-outcome-detail-title">
+        <%= t("consultation.final_outcome_detail") %>
+      </h1>
+    </div>
+    <div class="column-two-thirds" aria-labelledby="final-outcome-detail-title">
+      <%= render 'govuk_component/govspeak',
+          content: @content_item.final_outcome_detail,
+          direction: page_text_direction %>
+    </div>
+  </div>
+<% end %>
+
+<% if @content_item.public_feedback_documents? %>
+  <div class="grid-row sidebar-with-body">
+    <div class="column-third">
+      <h1 class="section-title" id="public-feedback-documents-title">
+        <%= t("consultation.feedback_received") %>
+      </h1>
+    </div>
+    <div class="column-two-thirds" aria-labelledby="public-feedback-documents-title">
+      <%= render 'govuk_component/govspeak',
+          content: @content_item.public_feedback_documents,
+          direction: page_text_direction %>
+    </div>
+  </div>
+<% end %>
+
+<% if @content_item.public_feedback_detail %>
+  <div class="grid-row sidebar-with-body">
+    <div class="column-third">
+      <h1 class="section-title" id="public-feedback-detail-title">
+        <%= t("consultation.public_feedback_detail") %>
+      </h1>
+    </div>
+    <div class="column-two-thirds" aria-labelledby="public-feedback-detail-title">
+      <%= render 'govuk_component/govspeak',
+          content: @content_item.public_feedback_detail,
+          direction: page_text_direction %>
+    </div>
+  </div>
+<% end %>
+
+<% if @content_item.final_outcome? %>
+  <section class="original-consultation">
+    <h2 class="section-title">Original consultation</h2>
+<% end %>
+
+<div class="consultation-banner">
+  <div class="grid-row">
+    <div class="column-third consultation-dates">
+      <% if @content_item.closed? %>
+        <p>
```

Comment:
You don't need to duplicate the `<p></p>` tags as they remain consistent for each case.